src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -0,0 +1,28 @@
+﻿#include "pch.h"
+#include "TerminalPage.h"
+#if __has_include("TerminalPage.g.cpp")
```

Comment:
y tho?

Reply:
this was autogenerated by the c++/winrt xaml file template actually

Reply:
presumably to support building with & without `CppWinRTOptimized`